charts/posthog/values.yaml

```diff
@@ -773,6 +773,52 @@ prometheus-kafka-exporter:
   kafkaServer:
     - posthog-posthog-kafka:9092
 
+
+###
+###
+### ---- prometheus-postgres-exporter ----
+###
+###
+prometheus-postgres-exporter:
+  # -- Whether to install the `prometheus-postgres-exporter` or not.
+  enabled: false
+
+  # -- Map of annotations to add to the pods.
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9187"
+
+  # -- Configuration options.
+  config:
+    datasource:
+      host: posthog-posthog-postgresql
+      user: postgres
+      passwordSecret: {
```

Comment:
non-blocking 

What's with the `{}` btw?

Reply:
It is valid YAML but I can remove it if you want.